### ISSUE TITLE
Pretrained _conv_stem.weight different each run when in_channels=1

### ISSUE THREAD

**Issue by daniel-a-diaz** [lukemelas/EfficientNet-PyTorch#330]
Just trying to understand what happens when in_channels=1. Every time I rerun the script the _conv_stem.weight is different each time. When in_channels=3 it is the same each time. It seems like something to do with padding. I am working on converting the 2D EfficientNet pretrained weights to 3D weights for use in https://github.com/shijianjian/EfficientNet-PyTorch-3D and this is hanging me up a bit. Thanks. I am using the below code to instantiate the model.

`model = EfficientNet.from_pretrained("efficientnet-b0", in_channels=1)`
